ui/app/components/secret-engine/list.hbs

```diff
@@ -49,19 +49,9 @@
       <div class="has-text-grey is-grid align-items-center linked-block-title">
         {{#if backend.icon}}
           <Hds::TooltipButton
-            @text={{if
-              backend.isSupportedBackend
-              (concat
-                (get (engines-display-data backend.type) "displayName")
-                (if
-                  (eq backend.version 2)
-                  " version 2"
-                  (if (and (eq backend.version 1) (eq backend.type "kv")) " version 1" "")
-                )
-              )
-              "The UI only supports configuration views for these secret engines. The CLI must be used to manage other engine resources."
-            }}
             aria-label="Type of backend"
+            @text={{this.tooltip}}
+            {{on "mouseover" (fn this.tooltipText backend)}}
```

Comment:
this seems like a good use of a [local helper function](https://guides.emberjs.com/release/components/helper-functions/#toc_local-helper-functions), that would remove the need for manually tracking `this.tooltip` and instead return text based on the `backend` passed in